slackbot-background/utils.go

```diff
@@ -86,3 +92,24 @@ func sendSlackCallback(msg *slack.Msg, responseUrl string) error {
 func isAlertConfirm(callbackId string) bool {
 	return strings.HasPrefix(callbackId, "alert_confirmation")
 }
+
+func deleteIpFromIprepd(ip string) error {
+	client := http.Client{Timeout: time.Second * 10}
+	for _, iprepdInstance := range config.IprepdInstances {
+		log.Infof("Sending DELETE request to %s for %s", iprepdInstance.URL, ip)
+
+		req, err := http.NewRequest("DELETE", iprepdInstance.URL+"/"+ip, nil)
```

Comment:
This is making use of the legacy API endpoint; might be good to update this to use the new one,

```go
req, err := http.NewRequest("DELETE", iprepdInstance.URL + "/type/ip/" + ip, nil)
```

Reply:
@ameihm0912 is iprepd 0.8 getting deployed within GCP here shortly? If not, happy to come back and update this once that is done.

Reply:
ahhh right, good call -- yeah let's leave as is then for now until we have coverage there